### PR TITLE
[blog] Add blog post about KubeVirt at KubeCon

### PR DESCRIPTION
**Pull request by codificat**
**What this PR does / why we need it**:

This adds a blog post to summarize KubeVirt's presence at the upcoming KubeCon in San Diego.

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: _posts/2019-11-12-kubecon-na-2019.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+comments: true
+title: KubeVirt at KubeCon + CloudNativeCon North America
+pub-date: Nov 12
+pub-year: 2019
+---
+
+The [KubeCon + CloudNativeCon North America 2019](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/)
+conference is next week in San Diego, California.
+
+KubeVirt will have presence at the event and this post highlights some
```
Comment (ptrnull) [inline _posts/2019-11-12-kubecon-na-2019.md]: have a presence

File: _posts/2019-11-12-kubecon-na-2019.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+activities that will have a KubeVirt focus there.
+
+### Sessions
+
+There are two sessisons that cover KubeVirt specifically:
```
Comment (ptrnull) [inline _posts/2019-11-12-kubecon-na-2019.md]: There are two sessions covering KubeVirt, specifically:

File: _posts/2019-11-12-kubecon-na-2019.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+- On Tuesday at 2:25 PM Chandrakanth Jakkidi and Steve Gordon will present an
+  [introduction to KubeVirt](https://sched.co/VyBC) that will cover the
+  background of the project, its motivation and use cases, and an architectural
+  overview and demo.
+
+- On Wednesday at 10:55 David Vossel and Vishesh Tanksale will take a deep-dive
```
Comment (ptrnull) [inline _posts/2019-11-12-kubecon-na-2019.md]: On Wednesday at 10:55 AM, David...

File: _posts/2019-11-12-kubecon-na-2019.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  will show KubeVirt's capabilities around host device passthrough using NVIDIA
+  GPU workloads as a case study.
+
+### Users and Contributors gathering
+
+KubeVirt users and contributors will get together to talk about KubeVirt,
+brainstorm ideas to help us shape the project’s next steps, and generally get
+some face to face time.
+
+If you are already using or contributing to KubeVirt, are considering to try it,
```
Comment (ptrnull) [inline _posts/2019-11-12-kubecon-na-2019.md]: If you are already using or contributing to KubeVirt, you are considering to try it,

**Comment by iranzo** [comment]
Preview at: https://deploy-preview-406--kubevirtio.netlify.com/2019/kubecon-na-2019.html

**Pull request merged**